.github/markdown-link-check-config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "timeout": "20s",
+    "retryOn429": true,
+    "retryCount": 5,
+    "fallbackRetryDelay": "30s",
+    "aliveStatusCodes": [
+        200,
+        206
+    ]
+}
```

Comment:
why the need for this config file in this repo but not in the relayer repo?

Reply:
There are more links in this repo and some edge case URLs being used. Given the higher failure rate the config helps with some flakey URLs (cough cough twittter)